root/static/scripts/edit/utility/guessFeat.js

```diff
@@ -114,9 +120,13 @@ function extractBracketedFeatCredits(str, artists, isProbablyClassical) {
   }, {name: str, joinPhrase: '', artistCredit: []});
 }
 
-function extractFeatCredits(
+export function extractFeatCredits(
   str, artists, isProbablyClassical, allowEmptyName,
 ) {
+  if (!featQuickTestRegex.test(str)) {
```

Comment:
I benchmarked `utils.similarTrackNames('Brooklyn Rumba', 'Brooklyn Rumba (feat. Dr. Israel & Marc Ribot)');` with and without this and it was only about 15% faster, so I don't think it would be noticeable when typing -- doesn't hurt, but can probably be removed.

Reply:
What is the point of having a "quick test"? Like, if it works quicker and is good shouldn't we always use it, and if it gives worse results wouldn't it be problematic anyway?

Reply:
Hmm, ok, I see. Well, I'm fine with leaving it in then if you feel the 15% improvement is significant enough - if not, I'd probably just use the old regex to limit complexity.